docs/source/installation.rst

```diff
@@ -129,3 +129,29 @@ First check whether the package is correctly installed. Do this with the command
 .. code-block:: bash
 
   python -m asreview oracle
+  
+
+Build dependencies error
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The command line returns the following message:
+
+.. code:: bash
+
+  "Installing build dependencies ... error"
+
+Some of the dependencies of ASReview are not compatible with your Python 
+installation. This error typically happens when the version of your Python 
+installation is either too old or too new. Then, the dependencies do not support 
```

Comment:
If the Python executable is not supported because it is too old, you will install an older version of ASReview automatically. So no error will pop up. This happens:

```
jonathan$ pip install asreview
Collecting asreview
  Downloading https://files.pythonhosted.org/packages/32/4e/808054f480f4ebab50b7657d02af49b77d46726adf569c71431bafca9350/asreview-0.3.0.tar.gz (66kB)
    100% |████████████████████████████████| 71kB 882kB/s 
Requirement already satisfied: numpy in /Users/jonathan/.pyenv/versions/anaconda2-2018.12/lib/python2.7/site-packages (from asreview) (1.15.4)
```

I think this might be confusing, I will yank those very old versions and see what kind of error pops up in that case.

Reply:
The behavior is slightly different as expected (and described in https://www.python.org/dev/peps/pep-0592/ ?). 

```
jonathan$ pip install asreview
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
WARNING: The candidate selected for download or install is a yanked version: 'asreview' candidate (version 0.3.0 at https://files.pythonhosted.org/packages/32/4e/808054f480f4ebab50b7657d02af49b77d46726adf569c71431bafca9350/asreview-0.3.0.tar.gz#sha256=ddd5cce2313faed924835eb1cca1a7b84805f2024ce89aefa7b3cc73c5078b9c (from https://pypi.org/simple/asreview/))
Reason for being yanked: <none given>
Collecting asreview
  Using cached asreview-0.3.0.tar.gz (66 kB)
```

The warning is raised in yellow btw. Maybe we should delete those very old versions of ASReview from Pypi (<=0.3). From that moment on, ASReview requires at least Python 3.6.

Reply:
https://github.com/asreview/asreview/pull/133

Reply:
We decided to delete the <0.4 releases. These releases are very unstable and users with a Pytohn <3.6 install might not notice that there try to install asreview==0.3.0 instead of the latest version. The error message will be:
```
jonathan$ pip install asreview
ERROR: Could not find a version that satisfies the requirement asreview (from versions: none)
ERROR: No matching distribution found for asreview
```

Reply:
Happy to merge this PR after removing the part on outdated Python distributions.

Reply:
Thanks! Deleting the <0.4 releases sounds like a good approach.